bookdown/04-technical/04-technical.Rmd

```diff
@@ -2,12 +2,39 @@
 
 This chapter gives an overview about technical details of the `r mlr_pkg("mlr3")` framework.
 
+**Parallelization**
+
 At first, some details about [Parallelization](#parallelization) and the usage of the `r cran_pkg("future")` are given.
-Parallelization refers to running multiple jobs simultaneously and is used to minimize necessary computer power. 
+Parallelization refers to the process of running multiple jobs simultaneously.
+Parallelization is used to minimize the necessary computer power.
+Irrespective, parallelization does not always minimize computer power.
```

Comment:
Is it called "computer power"? same one line above

Reply:
Good point.
Would computational power / computational capacity / computational energy fit better?

Reply:
I've never head this terms before in that context. Usually people use "computing power" or "computing resources". What do you think?